composer.json

```diff
@@ -44,10 +44,12 @@
         "dantleech/glob-finder": "^1.0",
         "jackalope/jackalope-jackrabbit": "^1.3",
         "matthiasnoback/symfony-dependency-injection-test": "^1.1",
+        "phpcr/phpcr": "^2.1",
+        "phpcr/phpcr-utils": "^1.4.0",
         "sonata-project/core-bundle": "^3.8",
         "symfony-cmf/resource": "^1.0",
         "symfony-cmf/resource-bundle": "^1.0",
-        "symfony-cmf/testing": "^2.0.1",
+        "symfony-cmf/testing": "2.0.*",
```

Comment:
shouldn't you keep `^`?

Reply:
A new `2.1.0` version of `symfony-cmf/testing` was released. And travis is failing with `^2.0.1` constraint, see https://travis-ci.org/sonata-project/SonataDoctrinePhpcrAdminBundle/builds/395047521 - i manually triggered that build. After #521 it should be updated back to `^2.1`

I want to propose changes to `dev-kit` for #512, but i want to fix travis first

Reply:
when not moving, we should keep "old" testing version here